plugins/modules/metadata_choice.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2024, Network to Code (@networktocode) <info@networktocode.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: metadata_choice
+short_description: Create, update or delete metadata choices within Nautobot
+description:
+  - Creates, updates or removes metadata choices from Nautobot
+notes:
+  - Tags should be defined as a YAML list
+  - This should be ran with connection C(local) and hosts C(localhost)
+author:
+  - Travis Smith (@tsm1th)
+version_added: "5.5.0"
+extends_documentation_fragment:
+  - networktocode.nautobot.fragments.base
+options:
+  metadata_type:
+    description:
+      - The name of the metadata type
+    required: true
+    type: raw
```

Comment:
```suggestion
    type: string
```

Since you used `set(["name"])` for the allowed query params of `metadata_type`, we don't really need `raw` here since we only allow passing the name. Unless you think they will need to search by more attributes?